Content.Server/Station/Systems/StationSystem.cs

```diff
@@ -496,6 +496,22 @@ public HashSet<EntityUid> GetStationsSet()
 
         return stats;
     }
+    // Goobstation start
+    public HashSet<EntityUid> GetAllStationGrids()
+    {
+        // Collect all grids owned by stations
+        var grids = new HashSet<EntityUid>();
+
+        var query = EntityQueryEnumerator<StationDataComponent>();
+        while (query.MoveNext(out var uid, out var data))
```

Comment:
Discard unused value
```suggestion
        while (query.MoveNext(out _, out var data))
```